src/platform.ts

```diff
@@ -153,10 +153,27 @@ export class App {
   async copyTemplate() {
     await promisifyHooks(this.opts.beforeCopy, this.hookArgsWithOriginalResourcesAppDir);
 
-    await fs.copy(this.opts.dir, this.originalResourcesAppDir, {
-      filter: userPathFilter(this.opts),
-      dereference: this.opts.derefSymlinks,
-    });
+    if (this.opts.tmpdir === false) {
+      const filter = userPathFilter(this.opts);
+      const items = await fs.readdir(this.opts.dir);
+      await Promise.all(items.map(async item => {
+        const srcItem = path.join(this.opts.dir, item);
+        const destItem = path.join(this.originalResourcesAppDir, item);
+
+        const include = await filter(srcItem, destItem);
+        if (!include) return;
+
+        return fs.copy(srcItem, destItem, {
+          filter,
```

Comment:
```suggestion
```

Since the judgment has already been made before, there is no need to let fs.copy execute the judgment again.